packages/shorebird_cli/lib/src/commands/patch/patch_android_command.dart

```diff
@@ -118,108 +119,118 @@ If this option is not provided, the version number will be determined from the p
     var hasBuiltWithActiveFlutter = false;
 
     final projectRoot = shorebirdEnv.getShorebirdProjectRoot()!;
-    final bundleDirPath = p.join(
-      projectRoot.path,
-      'build',
-      'app',
-      'outputs',
-      'bundle',
-    );
-    final bundlePath = flavor != null
-        ? p.join(bundleDirPath, '${flavor}Release', 'app-$flavor-release.aab')
-        : p.join(bundleDirPath, 'release', 'app-release.aab');
-
-    final String releaseVersion;
-    final argReleaseVersion = results['release-version'] as String?;
-    if (argReleaseVersion != null) {
-      logger.detail('Using release version $argReleaseVersion from argument.');
-      releaseVersion = argReleaseVersion;
-    } else {
-      logger.detail('No release version provided. Determining from bundle.');
-      final flutterVersionString =
-          await shorebirdFlutter.getVersionAndRevision();
-      final buildProgress = logger.progress(
-        'Building patch with Flutter $flutterVersionString',
-      );
+
+    return runScoped(() async {
```

Comment:
I'm not clear why we're adding an extra runScoped here – it doesn't look like we're overriding any dependencies?